### PR TITLE
Updates to CruiseInformation.csv

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR includes:

- Updates to existing CUIDs based on R2R (see attached spreadsheet for reference)

- Added P13b glider recovery cruise: TS20200522,F/V Tenacious II,2020-05-22T06:00:00,2020-05-22T23:59:59,Pioneer 13b glider recoveries

- 
[CruiseInformation_update_2020-05-22_RT.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/4670267/CruiseInformation_update_2020-05-22_RT.xlsx)
Deleted Papa 8 cruise

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -1,54 +1,75 @@
...
-MV-1309,R/V Melville,2013-07-15T00:00:00,2013-07-30T23:59:59,Papa 1
-KN-214,R/V Knorr,2013-11-20T18:00:00,2013-11-26T14:00:00,Pioneer 1
+TN-299,R/V Thomas G. Thompson,2013-06-30T00:00:00,2013-08-23T23:59:59,Cabled (rvdata) #TO BE DELETED
+TN299,R/V Thomas G. Thompson,2013-06-30T00:00:00,2013-08-23T23:59:59,Cabled (rvdata)
+MV-1309,R/V Melville,2013-07-15T00:00:00,2013-07-30T23:59:59,Papa 1 #TO BE DELETED
+MV1309,R/V Melville,2013-07-15T00:00:00,2013-07-30T23:59:59,Papa 1
+KN-214,R/V Knorr,2013-11-20T18:00:00,2013-11-26T14:00:00,Pioneer 1 #TO BE DELETED
+KN214,R/V Knorr,2013-11-20T18:00:00,2013-11-26T14:00:00,Pioneer 1
 TY-1402,CCGS Tully,2014-02-20T00:00:00,2014-02-20T00:00:00,Papa Glider Operations
 OC1404B,R/V Oceanus,2014-04-15T00:00:00,2014-04-21T23:59:59,Endurance 1 (rvdata)
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:10]: This should be after KN217 to be chronological.

File: cruise/CruiseInformation.csv [hunk 2]
```diff
@@ -1,54 +1,75 @@
...
+KN224,R/V Knorr,2014-12-11T19:00:00,2014-12-16T23:59:59,Pioneer 3 leg 2
 WJ20141230,M/V Warren Jr.,2014-12-30T13:30:00,2014-12-31T12:00:00,Pioneer ISSM-1 buoy recovery
 SI-1501,M/V Scarlett Isabella,2015-01-10T00:00:00,2015-01-13T23:59:59,#TO BE DELETED
 SI20150110,M/V Scarlett Isabella,2015-01-10T00:00:00,2015-01-13T23:59:59,Pioneer 3a
 PS1501,R/V Pacific Storm,2015-01-21T20:28:00,2015-01-21T22:02:00,Endurance GL info
-AT-26-29,R/V Atlantis,2015-02-12T15:00:00,2015-03-04T23:59:59,Southern Ocean 1
+AT-26-29,R/V Atlantis,2015-02-12T15:00:00,2015-03-04T23:59:59,Southern Ocean 1 #TO BE DELETED
+AT26-29,R/V Atlantis,2015-02-12T15:00:00,2015-03-04T23:59:59,Southern Ocean 1
 EN552,R/V Endeavor,2015-03-01T00:00:00,2015-03-05T23:59:59,Pioneer glider cp_387 recovery
 MM20150327,R/V Maria S. Merian,2015-03-27T00:00:00,2015-03-27T23:59:59,Pioneer glider cp_336 recovery
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:37]: This should be after EK-1503 to be chronological.

File: cruise/CruiseInformation.csv [hunk 3]
```diff
@@ -200,7 +237,8 @@ EK20190515,R/V Elakha,2019-05-15T15:30:00,2019-05-15T22:00:00,Endurance glider d
 TS20190516,F/V Tenacious II,2019-05-16T12:00:00,2019-05-17T12:00:00,Pioneer 12a glider deployment
 AR35-03,R/V Neil Armstrong,2019-05-27T00:00:00,2019-06-20T23:59:59,Irminger Glider gi_363 Recovery - Niskine Process Cruise
 EK20190529,R/V Elakha,2019-05-29T16:22:00,2019-05-29T19:33:00,Endurance ce01issp_00013 deployment
-AT-42-12,R/V Atlantis,2019-06-02T00:00:00,2019-07-09T00:00:00,Cabled (rvdata) Visions 19
+AT-42-12,R/V Atlantis,2019-06-02T00:00:00,2019-07-09T00:00:00,Cabled (rvdata) Visions 19 #TO BE DELETED
+AT42-12,R/V Atlantis,2019-06-02T00:00:00,2019-07-09T00:00:00,Cabled (rvdata) Visions 19
 ML20190612,R/V Miss Linda,2019-06-12T13:00:00,2019-06-13T18:00:00,Endurance glider recovery
 WJ20190617,M/V Warren Jr.,2019-06-17T18:00:00,2019-06-20T00:04:00,Pioneer 12b glider and AUV deployment
 EK20190624,R/V Elakha,2019-06-24T14:55:00,2019-06-24T19:00:00,Endurance ce02shsp_00013 deployment
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:244]: MZ20190626 should be here to be chronological.

File: cruise/CruiseInformation.csv [hunk 4]
```diff
@@ -1,54 +1,75 @@
 CUID,ShipName,cruiseStartDateTime,cruiseStopDateTime,notes
 UNRECOVERED,N/A,2009-09-01T00:00:00,2040-09-30T00:00:00,this is a placeholder for platforms/instruments that are not recovered
-TN-299,R/V Thomas G. Thompson,2013-06-30T00:00:00,2013-08-23T23:59:59,Cabled (rvdata)
-MV-1309,R/V Melville,2013-07-15T00:00:00,2013-07-30T23:59:59,Papa 1
-KN-214,R/V Knorr,2013-11-20T18:00:00,2013-11-26T14:00:00,Pioneer 1
+TN-299,R/V Thomas G. Thompson,2013-06-30T00:00:00,2013-08-23T23:59:59,Cabled (rvdata) #TO BE DELETED
+TN299,R/V Thomas G. Thompson,2013-06-30T00:00:00,2013-08-23T23:59:59,Cabled (rvdata)
+MV-1309,R/V Melville,2013-07-15T00:00:00,2013-07-30T23:59:59,Papa 1 #TO BE DELETED
+MV1309,R/V Melville,2013-07-15T00:00:00,2013-07-30T23:59:59,Papa 1
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply cruise/CruiseInformation.csv:6]: done

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (rtravis-whoi) [reply cruise/CruiseInformation.csv:10]: done

File: cruise/CruiseInformation.csv [hunk 5]
```diff
@@ -1,54 +1,75 @@
...
 OC1404B,R/V Oceanus,2014-04-15T00:00:00,2014-04-21T23:59:59,Endurance 1 (rvdata)
-KN-217,R/V Knorr,2014-04-11T17:00:00,2014-04-19T10:30:00,Pioneer 2
-KN-218,R/V Knorr,2014-05-01T00:00:00,2014-05-12T23:59:59,Pioneer glider cp_387 recovery
-MV-1404,R/V Melville,2014-06-10T00:00:00,2014-06-29T23:59:59,Papa 2
+KN-217,R/V Knorr,2014-04-11T17:00:00,2014-04-19T10:30:00,Pioneer 2 #TO BE DELETED
+KN217,R/V Knorr,2014-04-11T17:00:00,2014-04-19T10:30:00,Pioneer 2
+KN-218,R/V Knorr,2014-05-01T00:00:00,2014-05-12T23:59:59,Pioneer glider cp_387 recovery #TO BE DELETED
+KN218,R/V Knorr,2014-05-01T00:00:00,2014-05-12T23:59:59,Pioneer glider cp_387 recovery
+MV-1404,R/V Melville,2014-06-10T00:00:00,2014-06-29T23:59:59,Papa 2 #TO BE DELETED
+MV1404,R/V Melville,2014-06-10T00:00:00,2014-06-29T23:59:59,Papa 2
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply cruise/CruiseInformation.csv:16]: done

File: cruise/CruiseInformation.csv [hunk 6]
```diff
@@ -1,54 +1,75 @@
...
-KN-222,R/V Knorr,2014-10-03T22:00:00,2014-10-11T11:00:00,Pioneer 3 Leg 1
+KN-221-4,R/V Knorr,2014-09-07T09:00:00,2014-09-28T23:59:59,Irminger 1 #TO BE DELETED
+KN221-04,R/V Knorr,2014-09-07T09:00:00,2014-09-28T23:59:59,Irminger 1
+KN-222,R/V Knorr,2014-10-03T22:00:00,2014-10-11T11:00:00,Pioneer 3 Leg 1 #TO BE DELETED
+KN222,R/V Knorr,2014-10-03T22:00:00,2014-10-11T11:00:00,Pioneer 3 Leg 1
 OC1410A,R/V Oceanus,2014-10-06T00:00:00,2014-10-10T23:59:59,Endurance 2 (rvdata)
 AQ20141030,R/V Aquila,2014-10-30T00:00:00,2014-10-31T23:59:59,Papa Glider gp_453 recovery
-KN-224,R/V Knorr,2014-12-11T19:00:00,2014-12-16T23:59:59,Pioneer 3 leg 2
+KN-224,R/V Knorr,2014-12-11T19:00:00,2014-12-16T23:59:59,Pioneer 3 leg 2 #TO BE DELETED
+KN224,R/V Knorr,2014-12-11T19:00:00,2014-12-16T23:59:59,Pioneer 3 leg 2
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply cruise/CruiseInformation.csv:29]: done

File: cruise/CruiseInformation.csv (hunk 2 above)
Comment (rtravis-whoi) [reply cruise/CruiseInformation.csv:37]: done

File: cruise/CruiseInformation.csv [hunk 7]
```diff
@@ -77,14 +103,18 @@ CT20160808,R/V Connecticut,2016-08-08T00:00:00,2016-08-10T23:59:59,Pioneer 6a
...
 AQ20160827,R/V Aquila,2016-08-27T00:00:00,2016-08-28T23:59:59,Papa Glider gp_523 recovery
-NBP-16-08,R/V Nathaniel B. Palmer,2016-09-07T00:00:00,2016-10-14T23:59:59,Southern Ocean glider gs_566 recovery
+NBP-16-08,R/V Nathaniel B. Palmer,2016-09-07T00:00:00,2016-10-14T23:59:59,Southern Ocean glider gs_566 recovery #TO BE DELETED
+NBP1608,R/V Nathaniel B. Palmer,2016-09-07T00:00:00,2016-10-14T23:59:59,Southern Ocean glider gs_566 recovery
 AT37-03,R/V Atlantis,2016-09-18T20:15:00,2016-10-03T09:30:00,Endurance 6
 AR-08A,R/V Neil Armstrong ,2016-09-28T00:15:00,2016-10-05T11:00:00,Pioneer 7 Leg 1 #TO BE DELETED
 AR-08B,R/V Neil Armstrong ,2016-10-10T00:00:00,2016-10-16T23:59:59,Pioneer 7 Leg 2 #TO BE DELETED
-AR-08,R/V Neil Armstrong ,2016-09-28T00:15:00,2016-10-16T23:59:59,Pioneer 7
+AR-08,R/V Neil Armstrong ,2016-09-28T00:15:00,2016-10-16T23:59:59,Pioneer 7 #TO BE DELETED
+AR8,R/V Neil Armstrong ,2016-09-28T00:15:00,2016-10-16T23:59:59,Pioneer 7
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply cruise/CruiseInformation.csv:112]: done

File: cruise/CruiseInformation.csv (hunk 3 above)
Comment (rtravis-whoi) [reply cruise/CruiseInformation.csv:244]: done

File: cruise/CruiseInformation.csv [hunk 8]
```diff
@@ -227,8 +269,8 @@ SU20200220,R/V Pacific Surveyor,2020-02-20T22:55:00,2020-02-20T14:50:00,Enduranc
 AR42-02,R/V Neil Armstrong,2020-02-24T12:00:00,2020-03-04T23:59:00,Pioneer glider cp_340 recovery
 TS20200301,F/V Tenacious II,2020-03-01T20:00:00,2020-03-02T15:00:00,Pioneer 13b glider deployments
 WJ20200323,M/V Warren Jr.,2020-03-23T00:00:00,2020-03-27T23:59:00,Pioneer 13c AUV deployments
-AR-45,R/V Neil Armstrong,2020-04-17T00:00:00,2020-05-07T23:59:00,Pioneer 14
 AJ20200506,R/V Adolf Jensen,2020-05-06T12:00:00,2020-05-08T12:00:00,Irminger glider gi_525 recovery
-SKQ202009S,R/V Sikuliaq,2020-05-21T00:00:00,2020-06-06T23:59:00,Papa 8
+TS20200522,F/V Tenacious II,2020-05-22T06:00:00,2020-05-22T23:59:59,Pioneer 13b glider recoveries
+AR44,R/V Neil Armstrong,2020-06-07T00:00:00,2020-06-17T23:59:00,Pioneer 14
 AR46-01,R/V Neil Armstrong,2020-05-22T00:00:00,2020-06-21T23:59:00,Irminger 7
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply cruise/CruiseInformation.csv:275]: done

File: vessel/shiplist.csv [hunk 9]
```diff
@@ -1,5 +1,5 @@
 Prefix,Vessel Designation,Vessel Name,ICES Code,Operator,Call Sign,MMSI#,IMO#,Length (m),Max Speed (m/s),Max Draft (m),Designation,Active,R2R
-AJ,R/V,Adolf Jensen,,60North,,,,,,,,Y,N
+AJ,R/V,Adolf Jensen,,60North,OYTL,331185000,6708575,,,,,Y,N
```
Comment (snwhite) [inline vessel/shiplist.csv:2]: Looks good.

File: cruise/CruiseInformation.csv [hunk 10]
```diff
@@ -227,8 +269,8 @@ SU20200220,R/V Pacific Surveyor,2020-02-20T22:55:00,2020-02-20T14:50:00,Enduranc
...
 WJ20200323,M/V Warren Jr.,2020-03-23T00:00:00,2020-03-27T23:59:00,Pioneer 13c AUV deployments
-AR-45,R/V Neil Armstrong,2020-04-17T00:00:00,2020-05-07T23:59:00,Pioneer 14
 AJ20200506,R/V Adolf Jensen,2020-05-06T12:00:00,2020-05-08T12:00:00,Irminger glider gi_525 recovery
-SKQ202009S,R/V Sikuliaq,2020-05-21T00:00:00,2020-06-06T23:59:00,Papa 8
-AR46-01,R/V Neil Armstrong,2020-05-22T00:00:00,2020-06-21T23:59:00,Irminger 7
-AR-48,R/V Neil Armstrong,2020-09-30T00:00:00,2020-10-20T23:59:00,Pioneer 15
+TS20200522,F/V Tenacious II,2020-05-22T06:00:00,2020-05-22T23:59:59,Pioneer 13b glider recoveries
+AR44,R/V Neil Armstrong,2020-06-07T00:00:00,2020-06-17T23:59:00,Pioneer 14
+AR46,R/V Neil Armstrong,2020-08-08T00:00:00,2020-09-06T23:59:00,Irminger 7
+AR48,R/V Neil Armstrong,2020-09-30T00:00:00,2020-10-20T23:59:00,Pioneer 15
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:276]: All updates look good.

File: vessel/shiplist.csv (hunk 9 above)
Comment (cdobs) [inline vessel/shiplist.csv:2]: Looks good

File: cruise/CruiseInformation.csv (hunk 4 above)
[the remark this answers is not in the document]
Comment (cdobs) [reply cruise/CruiseInformation.csv]: Looks good

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (cdobs) [reply cruise/CruiseInformation.csv]: Looks good

File: cruise/CruiseInformation.csv (hunk 5 above)
[the remark this answers is not in the document]
Comment (cdobs) [reply cruise/CruiseInformation.csv]: looks good

File: cruise/CruiseInformation.csv (hunk 6 above)
[the remark this answers is not in the document]
Comment (cdobs) [reply cruise/CruiseInformation.csv]: looks good

File: cruise/CruiseInformation.csv (hunk 2 above)
Comment (cdobs) [reply cruise/CruiseInformation.csv]: looks good

File: cruise/CruiseInformation.csv (hunk 7 above)
[the remark this answers is not in the document]
Comment (cdobs) [reply cruise/CruiseInformation.csv]: looks good

File: cruise/CruiseInformation.csv (hunk 3 above)
Comment (cdobs) [reply cruise/CruiseInformation.csv:244]: looks good

File: cruise/CruiseInformation.csv (hunk 10 above)
Comment (cdobs) [reply cruise/CruiseInformation.csv:276]: All look good

**Pull request merged**